src/main/java/com/cozyhome/onlineshop/userservice/security/JWT/JwtTokenFilter.java

```diff
@@ -32,7 +32,7 @@ public class JwtTokenFilter extends OncePerRequestFilter {
     @Value("${header.name.user-role}")
     private String userRoleAttributeName;
     @Value("${api.basePath}")
-    private String noAuthPathUrl;
+    private String basePathUrl;
```

Comment:
I'd leave it as it was. Because the name should reflect the meaning of the action in the method.

Reply:
"basePathUrl" doesn't describe the meaning of the method. I disagree.